WordPress/src/main/java/org/wordpress/android/ui/stats/StatsUtils.java

```diff
@@ -352,6 +352,18 @@ public static synchronized void logVolleyErrorDetails(final VolleyError volleyEr
         AppLog.e(T.STATS, "Volley Error details: " + volleyError.getMessage(), volleyError);
     }
 
+    public static synchronized  boolean isAuthenticationError(final VolleyError volleyError) {
+        if (volleyError != null && volleyError.networkResponse != null) {
```

Comment:
Your point is valid, and I even checked the Volley src to be sure AuthFailureError is thrown correctly. It is, obviously.

Although, I remember I had to do that ugly check above because there was something strange on the server side with jetpack and I needed to get the 403 error message with that particular string in it.  Maybe it's not necessary anymore.
Anyway, before spending other time on this PR, I want to be sure that it is useful.